saya/core/src/prover/atlantic/client.rs

```diff
@@ -1,17 +1,19 @@
 use std::{borrow::Cow, time::Duration};
 
-use anyhow::Result;
+use crate::prover::error::ProverError;
+use cairo_vm::types::layout_name::LayoutName;
 use reqwest::{
     multipart::{Form, Part},
     Client, ClientBuilder,
 };
 use serde::Deserialize;
 use url::Url;
 
-const ATLANTIC_API_BASE: &str = "https://atlantic.api.herodotus.cloud/v1";
+const ATLANTIC_API_BASE: &str = "https://staging.atlantic.api.herodotus.cloud";
+const ATLANTIC_S3_BASE: &str = "https://s3.pl-waw.scw.cloud/atlantic-k8s-experimental/queries";
```

Comment:
As a note for future PRs, Atlantic now returns the list of metadata with URL for each files. This will be better than hard code the s3 base, which may change in the future.